User_interface/user_interface.py

```diff
@@ -718,7 +718,7 @@ def visualise_2D(attribute_to_plot, file_to_plot, ax_canvas, text_box, frame, bo
     material_data_dict = json.loads(data)
 
     config_file = material_data_dict['config_file']
-    path = os.path.dirname(os.path.abspath(__file__)) + '/../Input_config_files/' + config_file
+    path = os.path.dirname(os.path.abspath(__file__)) + '/../Input_config_files/' + config_file[0]
```

Comment:
why [0]?

Reply:
When we were showing rickard at the delivery the visualization didn't work without this

Reply:
It's because the config file is saved in a list, so there's trouble converting it to a string